pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -207,10 +214,7 @@ func (q *Queue) Reconcile(ctx context.Context) (reconcile.Result, error) {
 		if !IsUnrecoverableError(err) {
 			// store the error that is causing us to fail, so we can bubble it up later if this times out.
 			cmd.lastError = err
-			// mark this item as done processing. This is necessary so that the RLI is able to add the item back in.
-			q.TypedRateLimitingInterface.Done(cmd)
-			q.TypedRateLimitingInterface.AddRateLimited(cmd)
-			return reconcile.Result{RequeueAfter: singleton.RequeueImmediately}, nil
+			return reconcile.Result{Requeue: true}, nil
```

Comment:
This is requeueing with backoff -- I almost think that we should try to requeue with a consistent poll because we are almost always going to hit the maxBackoff given that it's going to take around a minute for the nodes to come up